Apps/W1/Shopify/app/src/Order handling/Codeunits/ShpfyImportOrder.Codeunit.al

```diff
@@ -299,6 +299,25 @@ codeunit 30161 "Shpfy Import Order"
         end;
     end;
 
+    [NonDebuggable]
+    local procedure ImportCustomAttributtes(ShopifyOrderId: BigInteger; OrderLineId: Guid; JCustomAttributtes: JsonArray)
+    var
+        OrderLineAttribute: Record "Shpfy Order Line Attribute";
+        JToken: JsonToken;
+    begin
+        OrderLineAttribute.SetRange("Order Id", ShopifyOrderId);
+        if not OrderLineAttribute.IsEmpty then
+            OrderLineAttribute.DeleteAll();
+        foreach JToken in JCustomAttributtes do begin
+            Clear(OrderLineAttribute);
```

Comment:
I don't know how microsoft is working with this, but we in our company use always init and Insert(true), despite in for special scenarios (temporary tables..)